draft-ietf-rats-reference-interaction-models.md

```diff
@@ -160,11 +160,15 @@ A PKIX Certificate is an X.509v3 certificate as specified by {{RFC5280}}.
 In the context of this document, the term "Remote" does not necessarily refer to a remote entity in the scope of network topologies or the Internet.
 It rather refers to decoupled systems or entities that exchange the Conceptual Message type called Evidence {{-RATS}}.
 This conveyance can also be "Local", if the Verifier role is part of the same entity as the Attester role, e.g., separate system components of the same Composite Device (a single RATS entity), or the Verifier and Relying Party roles are hosted by the same entity, for example in a cryptographic key broker system.
-Even if an entity takes on two or more different roles, the functions they provide typically reside in isolated environments that are components of the same entity. Examples of such isolated environments include a Trusted Execution Environment (TEE), Baseboard Management Controllers (BMCs), as well as other physical or logical protected/isolated/shielded Computing Environments (e.g., embedded Secure Elements (eSE) or Trusted Platform Modules (TPM)). Readers of this document should be familiar with the concept of Layered Attestation as described in {{Section 3.1 of {{-RATS}} and the definition of Attestation as described in {{-RIV}}.
+Even if an entity takes on two or more different roles, the functions they provide typically reside in isolated environments that are components of the same entity. Examples of such isolated environments include a Trusted Execution Environment (TEE), Baseboard Management Controllers (BMCs), as well as other physical or logical protected/isolated/shielded Computing Environments (e.g., embedded Secure Elements (eSE) or Trusted Platform Modules (TPM)). It is useful but not necessary for readers of this document to be familiar with the Conceptual Data/Messages flows as described in {{Section 3 of {{-RATS}} and the definition of Attestation in general as described in {{-RIV}}.
 
 # Scope and Intent
 
-This document outlines the interaction models between Attesters and Verifiers to convey Evidence.
+This document outlines three very common interaction models between RATS roles.
+In this document the interaction models illustrated focus on the conveyance of Evidence about boot-time integrity from Attesters to Verifiers.
```

Comment:
(same comment I made on the email reply)

Unsure why run-time measurements would be ruled out.  The dynamicity and kind of the target environment seem orthogonal to the conveyance of associated evidence.

Reply:
> The dynamicity and kind of the target environment seem orthogonal to the conveyance of associated evidence.

I am not sure what you are eluding to, tbh. Could you help us and elaborate a bit?

Reply:
I am OK content-wise, but the phrasing is a bit awkward.  My humble editorial suggestion:

> The document illustrates the interaction models focusing on conveying Evidence about boot-time integrity from Attesters to Verifiers.
> It does not exclude application to runtime integrity or other conceptual messages, but these are not illustrated.

Reply:
What I meant is that in the context of a framework that aims at describing the conveyance of CMs, it does seem odd to impose restrictions based on the internal details of one or more CMs.  IOW, from the point of view of a conveyance protocol, Evidence is Evidence, be it runtime or boottime, dynamic or static.

Reply:
Please see the updated suggestion above. Does that work for you?